benefits/core/migrations/local_fixtures.json

```diff
@@ -185,101 +145,36 @@
     "model": "core.eligibilityverifier",
     "pk": 3,
     "fields": {
-      "name": "(MST) eligibility server verifier",
+      "name": "(CST) eligibility server verifier",
```

Comment:
(Not for this PR): eventually we should have a new `core.authprovider` instance for this verifier:

```json
{
  "model": "core.authprovider",
  "pk": 3,
  "fields": {
    "sign_out_button_template": "core/includes/button--sign-out--login-gov.html",
    "sign_out_link_template": "core/includes/link--sign-out--login-gov.html",
    "client_name": "calfresh-benefits-oauth-client-name",
    "client_id_secret_name": "auth-provider-client-id",
    "authority": "https://example.com",
    "scope": "verify:calfresh",
    "claim": "calfresh",
    "scheme": "dev-cal-itp_benefits"
  }
},
```

Then update the field here on the `eligibilityverifier`:

```diff
- "auth_provider": 1
+ "auth_provider": 3
```